java-checks/src/test/files/checks/EmptyStatementUsageCheck.java

```diff
@@ -18,7 +18,7 @@ public Foo() {
 
   void foo(boolean condition) {
     for (int i = 0; i < 42; i++)
-      ; // Noncompliant {{Remove this empty statement.}}
+      ; // compliant unique statement of a loop
```

Comment:
@benzonico we probably need to update the RSPEC as well then, because similar case (but with multiple incrementors) was explicitly defined: https://jira.sonarsource.com/browse/RSPEC-3203

These cases won't raise issues anymore.

Reply:
done